### ISSUE TITLE
Binds not working and restarting service raises secure keyboard entry error

### ISSUE THREAD

**Issue by dmfay** [koekeishiya/khd#18]
Environment: OSX Sierra 10.12, zsh; I usually use iTerm2 but have been trying this in Terminal.app just in case. I installed kwm 4.0.2 and khd 1.1.0 through Homebrew and started/enabled them with `brew services`.  I do not have secure keyboard entry enabled, and kwm and khd are both allowed to control the computer. My configuration is copied verbatim from the kwm examples (including khdrc). 

kwm is running; the border is present and follows mouse focus as expected. When I first boot, khd is also running, /tmp/khd.err is empty, and /tmp/khd.out shows it processing the config file. It does cut off suddenly a bit over halfway through, although I'm not sure if that means anything:

```
Khd: Using config '/Users/dianfay/.khdrc'
Token_Comment:  Enable kwm compatibility mode
Token_Comment:  set color of default mode
Prefix Color: 0xFFD5C4A1
...
Token_Modifier: cmd
Token_Modifier: alt
Token_Literal: 3
Token_Command: kwmc display -f 2
Token_Comment:  Swap Focused Window
Token_Modifier: ctrl
Token_Modifier% 
```

The problem is that none of the binds in khdrc seem to register. I've been trying cmd+ctrl+r to rotate windows and cmd+alt+vi keys to move focus to no avail (although cmd+alt+h is hiding other windows in the current desktop). No further output is produced in either khd.out or khd.err. I did `sudo launchctl config user path /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin` from #9 since I'm using an alternative shell, but it does not appear to have affected anything.

The kicker: when I `brew services restart khd`, `Khd: Secure keyboard entry is enabled! Terminating.` appears in khd.err, sometimes multiple times. khd.out is empty.

**Comment by f1337** [comment]
I had the same issue. I was able to fix by restarting, then enabling `khd` in System Preferences > Security & Privacy > Privacy > Accessibility. I made sure that `khd` was enabled with `brew services start khd` before restarting.

**Comment by dmfay** [comment]
That hasn't worked for me, unfortunately. I gave it another try, removing khd from the Accessibility preferences and restarting; when it came back up khd requested access and I granted it, but it failed to start and dumped several secure keyboard entry warnings into the error file. Restarting again returned it to the state where khd is running but not catching key events, with the output file again cutting off at the same place.
